tests/unit/tasks/server/livereload-server-test.js

```diff
@@ -100,7 +100,9 @@ describe('livereload-server', function () {
           port: 4200,
         })
         .then(function () {
-          http.get('http://localhost:4200/_lr/livereload.js', function (response) {
+          let httpGetOptions = { headers: { Connection: 'close' } };
+
+          http.get('http://localhost:4200/_lr/livereload.js', httpGetOptions, function (response) {
```

Comment:
I'm unsure if this is the right fix, but on Node.js v20, this `GET` request is being sent with `Connection: keep-alive` as opposed to `Connection: close` on older Node.js versions. `Connection: keep-alive` causes the `done` callback below to not being called anymore.

Reply:
was there ever a problem with `ember test`? or was it just the tests here?

Reply:
I assume it's just this test. I think if `ember test` or any other command didn't work on Node.js v20, we would've gotten more bug reports already. Just a guess though!